README.md

```diff
@@ -112,3 +112,62 @@ Add a listener for request
 ```
 client.eventSource.on('requests', function (err, res))
 ```
+
+
+### Managing Child/Related Devices
+
+Related Device accessor
```

Comment:
nit: These look like they can be level 4 headers.

Reply:
what are those ?

Reply:
I think he just means go deeper `####` vs `###`

Reply:
Prolly a good idea